newsilkroad/urls.py

```diff
@@ -29,6 +29,7 @@
     url(r'^', include('website.urls')),
     url(r'^filer/', include('filer.urls')),
     url(r'^api/', include('api.urls')),
+    url(r'^infrastructure/', include('infrastructure.urls')),
```

Comment:
These are included under  `/database/` in `website/urls.py` already. We shouldn't need to add them again.